Lection 3/task 3.1/ReportApp/Models/Activity/ActivityReportConfiguration.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ReportApp.Models;
+
+public class ActivityReportConfiguration : ReportConfiguration
+{
+    public string SavePath { get; } = "../../../Reports/ActivityReport.xlsx";
```

Comment:
Why are we making it as a property? I showed you an example how it should works, so why?